docs/api/config.md

```diff
@@ -495,7 +558,17 @@ Each pattern can start with a variable that resolves to a system base directory.
 Can be any of the following types:
 
 - `string[]`: A list of paths that are allowed by this scope.
-- `object`: A complete scope configuration.
+- `{ "allow": [string], "deny": [string], "requireLiteralLeadingDot": boolean? }`: A complete scope configuration.
+
+	
+	| Name | Type | Default | Description |
+	| ---- | ---- | ------- | ----------- |
+	| <div className="anchor-with-padding" id="fsallowlistscope.allow">`allow`<a class="hash-link" href="#fsallowlistscope.allow"></a></div> | [string] | [] | A list of paths that are allowed by this scope. |
```

Comment:
the type format is incorrect, it should be

```suggestion
	| <div className="anchor-with-padding" id="fsallowlistscope.allow">`allow`<a class="hash-link" href="#fsallowlistscope.allow"></a></div> | string[] | [] | A list of paths that are allowed by this scope. |
```

Reply:
I've fixed it via the generator script.